### ISSUE TITLE
[BUG] Facebook page connector publishing twite

### ISSUE THREAD

**Issue by friscoMad** [kvangent/PokeAlarm#355]
## Problem Description
PokeAlarm is publishing twice for each alarm one with the link and another without it.
![pokemad_2017-03-19_13-19-14](https://cloud.githubusercontent.com/assets/487098/24080816/b422bba4-0ca6-11e7-8c2d-9de72204c025.png)

## Your Environment
Operating System : Centos 7
Python Version: Python 2.7.5
PokemonGo-Map startup command `cmd`: python run_server.py
PokeAlarm startup command `cmd` python start_pokearlarm.py

Did the issue persist after restarting the script: yes

## Console Output
Nothing special

## Your Alarms Config
```json
    [
	{
		"active": "True",
		"type": "pushbullet",
		"startup_message":"False",
		"api_key": "o.UP2cKvvNMB4",
		"pokemon":{
			"channel":"pokemad",
			"title":"¡Un <pkmn> salvaje apareció!",
			"url":"<gmaps>",
			"body":"Huye en <time_left> (<24h_time>).\nDirección: <address>"
		}
	}, 
	{
		"active": "True",
		"type":"discord",
		"webhook_url":"https://discordapp.com/api/webhooks/2929
		"startup_message":"False",
		"pokemon":{
			"webhook_url":"https://discordapp.com/apifskI-IUatwmz9DZMkN",
			"username":"<pkmn>",
			"icon_url" : "https://map.neoseo.es/core/pokemons/<pkmn_id>_flat.png",
			"title":"¡Un <pkmn> salvaje apareció!",
			"url":"<gmaps>",
			"body": "Dirección: <address>.\nHuye en <time_left> (<24h_time>).",
			"map": {             
				"enabled":"true", 
				"width":"250",    
				"height":"125",  
				"maptype":"roadmap",
				"zoom": "15"      
			}                      
		}		
	},
    {
		"active": "True",
		"type": "twitter",
		"startup_message":"False",
		"access_token": "8433789060612PFxMsKSl8WDi0",
		"access_secret": "vmR4f5LKwne8tgcccmJ",
		"consumer_key": "R4j1X4kA",
		"consumer_secret": "IK0EShrQOgidL",
		"pokemon":{
			"status": "¡Un <pkmn> salvaje apareció! Huye en <time_left> (<24h_time>). <gmaps> #PokemonGo #Madrid"
		}
	},
	{
		"active": "True",
		"type":"facebook_page",
		"page_access_token":"EAATX8fiiwZDZD",
		"startup_message":"False",
		"pokemon":{
			"message": "¡Un <pkmn> salvaje apareció!\nHuye en <time_left> (<24h_time>).\nDirección: <address>\n#PokemonGo #Madrid",
			"link": "<gmaps>"
		}
	}	
]
```

## Troubleshooting Checklist
[X ] Most recent version of PokemonGo-Map
[X ] Most recent version of PokeAlarm
[X ] Checked the Wiki page for service in question

**Comment by kvangent** [comment]
There is already a ticket open for this (#354) and a fixed posted in commit 35d8e359e2eb431b88fefa0506b9ebc582343acd.

**Comment by friscoMad** [comment]
Lol 
Sorry I started to write the issue this morning but needed to go and did not publish it till now.
